docs/v0.5/getting-started/docker-for-mac.md

```diff
@@ -71,27 +71,32 @@ watch -n 1 kubectl get pod --all-namespaces
 
 Load the projectriff charts
 
-```ah
+```sh
 helm repo add projectriff https://projectriff.storage.googleapis.com/charts/releases
 helm repo update
 ```
 
-riff can be installed with or without Knative. The riff [Core runtime](../runtimes/core.md) is available in both environments, however, the riff [Knative Runtime](../runtimes/knative.md) is only available if Knative is installed.
+riff can be installed with optional runtimes. The riff build system is always installed, and is required by each runtime.
 
-To install riff with Knative and Istio:
+If using the Knative runtime, first install Istio:
 
 ```sh
-helm install projectriff/istio --name istio --namespace istio-system --set gateways.istio-ingressgateway.type=NodePort --wait
-helm install projectriff/riff --name riff --set knative.enabled=true
+helm install projectriff/istio --name istio --namespace istio-system --set gateways.istio-ingressgateway.type=NodePort --wait --devel
 ```
 
-Alternatively, install riff without Knative or Istio:
+Install riff with the desired runtime (pick any, all or none):
+
+Append:
+
+- `--set riff.runtimes.core.enabled=true` to enable the Core runtime
+- `--set riff.runtimes.knative.enabled=true` to enable the Knative runtime
+- `--set riff.runtimes.streaming.enabled=true` to enable the Streaming runtime
 
 ```sh
-helm install projectriff/riff --name riff
+helm install projectriff/riff --name riff --wait --devel
 ```
```

Comment:
How about:

> Install riff with all 3 available runtimes. (To install just one or more runtimes, replace `true` with `false` on the relevant lines below.)
> 
> ```sh
> helm install projectriff/riff --name riff \
>   --set riff.runtimes.core.enabled=true \
>   --set riff.runtimes.knative.enabled=true \
>   --set riff.runtimes.streaming.enabled=true \
>   --wait --devel
> ```

Reply:
the streaming runtime has additional requirements and while fail with these instructions as is. Perhaps we should remove it, flip the default to `false` for each runtime to require opt-in rather than opt-out.

Reply:
> the streaming runtime has additional requirements

is it possible to document installing the bits that streaming needs first?

Reply:
> is it possible to document installing the bits that streaming needs first?

@ericbottard are we ready for that?

Reply:
We're ready to document that, sure. We basically need to install keda via helm, or perform the equivalent of that in our chart when the streaming.enabled flag is set. I'm not sure if that is possible, both from a technical and a license perspective?

Reply:
OK, so the latest stance on this is that
- we're limited to helm 2.13.x due to https://github.com/helm/helm/issues/5806
- can't install both knative and streaming at the same due to another limitation of helm

Reply:
@scothis if you're ok with the suggested change above (applied across all 4 getting started pages, can we merge this?

Reply:
I'd still lean towards disabling all runtimes by default with opt-in to the additional bits. Especially since the Knative runtime has a pre-req (Istio)

Reply:
perhaps we don't include the streaming runtime setting for now. We can add it along with other docs for the streaming runtime